README.md

```diff
@@ -62,7 +62,7 @@ Contributions, issues, and feature requests are welcome!
 
 Give a ⭐ if you like this project and how it is built!
 
-## 📝 License
 
-This project is [MIT](https://github.com/Smart1-hub/Smart Development/blob/add-license-1/LICENSE)
+## 📝 License
 
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Hey, Great job on implementing the changes required by the previous review, however, you made the changes but you've added a broken link. Kindly make your MIT.md a link to this [microverse MIT.md file](https://github.com/microverseinc/readme-template/blob/master/MIT.md)